### ISSUE TITLE
Destroyed walls appear as intact in the room description

### ISSUE THREAD

**Issue by JimViebke** [JimViebke/IslandMUD-server#23]
*"This room consists of a(n) stone wall to the north..."*

**"a(n) stone wall to the north"** should read **"a pile of rubble to the north"**.

**Comment by JimViebke** [comment]
Duplicate of #27

**Comment by JimViebke** [comment]
Not a dupliate of #27, just a bad title. Re-opening and renaming.

**Comment by JimViebke** [comment]
The issue name was in error. The issue name has been edited to refer to the issue in the description, which has been resolved.

Updates will be pushed to binaries once milestone is complete.
